rpm/usb-moded.spec

```diff
@@ -117,6 +117,20 @@ system bus.
 This package contains the diag config for use with the android
 gadget driver.
 
+%package diag-mode-androidv5+qcom
```

Comment:
Do not use + in the package name as it might break. Also I would leave the .ini files out from this for now as it causes change of functionality to existing devices. Lets see how things start forming and then consider how and where put the .ini file changes.